test/integration_run_stack_test.go

```diff
@@ -0,0 +1,50 @@
+package test_test
```

Comment:
Consider renaming the file to `integration_run_test.go` for clarity.
This would also leave room to add tests for other `run` commands in the future.